<launch>
  <arg name="max_acceleration" default="1.0" />
  <arg name="max_velocity" default="2.0" />
  <arg name="distance" default="5.0" />
  
  <node name="motion_profile_visualiser" 
        pkg="test_pkg" 
        type="MotionProfileVis"
        output="screen">
    <param name="max_acceleration" value="$(arg max_acceleration)" />
    <param name="max_velocity" value="$(arg max_velocity)" />
    <param name="distance" value="$(arg distance)" />
    <param name="time_step" value="0.01" />
  </node>
  
  <node name="rqt_plot" 
        pkg="rqt_plot" 
        type="rqt_plot" 
        args="/position/data /velocity/data /acceleration/data" />
</launch>